src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -3,16 +3,28 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
```

Comment:
@DavidKarlas @bozturkMSFT could you please review public API for template package management in IDE? 

I'm still working on tests, however would appreciate early review to see in case anything is missing in API. 
Note that IDE idea is to provide simple API for host developers. It is not supposed to cover all the features; if host developer needs advanced features they might use the `TemplateEngine.Edge` directly.